src/components/Note/Note.scss

```
@@ -53,37 +53,37 @@ $note__author-image-size: 26px;
   outline: 2px solid rgba(var(--accent-color-rgb), 0.5);
 }
 
-.note:hover {
+// Only applies hover transform while not dragging
+body:not(:has(> div.drag-overlay)) .note:hover {
```

I know I did this, but if you have a better version of it, be my guest.
Wanted to apply `.note:hover` only if body has no child with drag overlay

---

Nevermind, due to the 0.98 scale on combine, this is useless.
```suggestion
.note:hover {
```